web/src/app/admin/skills/skills.component.ts

```diff
@@ -1,14 +1,105 @@
-import {Component, OnInit} from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { SkillsService } from './shared/skills.serviceApp';
+import { SkillViewModel } from '../../webapi/models/skill-view-model';
 
 @Component({
   selector: 'skills',
-  templateUrl: 'skills.component.html'
+  templateUrl: 'skills.component.html',
+  styleUrls: ['./skills.component.css']
 })
 
 export class SkillsComponent implements OnInit {
-  constructor() {
-  }
 
+  skillsModel: SkillViewModel = {} as SkillViewModel;
+  skills: SkillViewModel[] = [];
+  skillsList: SkillViewModel[];
```

Comment:
what is the purpose of two list variables

Reply:
done